sdk/include/psdk/rpc.h

```diff
@@ -122,25 +122,14 @@ typedef int RPC_STATUS;
     #include <excpt.h>
     #include <winerror.h>
 
-    #ifndef __USE_PSEH2__
-        #define RpcTryExcept __try {
-        #define RpcExcept(expr) } __except (expr) {
-        #define RpcEndExcept }
-        #define RpcTryFinally __try {
-        #define RpcFinally } __finally {
-        #define RpcEndFinally }
-        #define RpcExceptionCode() GetExceptionCode()
-        #define RpcAbnormalTermination() AbnormalTermination()
-    #else
-        #define RpcTryExcept _SEH2_TRY
-        #define RpcExcept(expr) _SEH2_EXCEPT((expr))
-        #define RpcEndExcept _SEH2_END;
-        #define RpcTryFinally _SEH2_TRY
-        #define RpcFinally _SEH2_FINALLY
-        #define RpcEndFinally _SEH2_END;
-        #define RpcExceptionCode() _SEH2_GetExceptionCode()
-        #define RpcAbnormalTermination() (_SEH2_GetExceptionCode() != 0)
-    #endif
+    #define RpcTryExcept _SEH2_TRY
+    #define RpcExcept(expr) _SEH2_EXCEPT((expr))
+    #define RpcEndExcept _SEH2_END;
+    #define RpcTryFinally _SEH2_TRY
+    #define RpcFinally _SEH2_FINALLY
+    #define RpcEndFinally _SEH2_END;
+    #define RpcExceptionCode() _SEH2_GetExceptionCode()
+    #define RpcAbnormalTermination() (_SEH2_GetExceptionCode() != 0)
```

Comment:
OK but this makes our hypothetical "MS-PSDK-compatible headers" now unconditionally depend on ReactOS-specific stuff: the PSEH macros.

Reply:
OK, I restored the old code and moved the `__USE_PSEH2__` definition to pseh2.h. It will be included through excpt.h